.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!-- This form is for reporting bugs and issues specific to the Paid Memberships Pro plugin. This is not a support portal. If you need technical support from a human being, please submit a ticket via the helpdesk instead: https://www.paidmembershipspro.com/support/ -->
```

Comment:
I don't think we need this file if we have the others defined

Reply:
I will do another PR to remove it.